node/packages/parser/package.json

```diff
@@ -17,6 +17,7 @@
     "@shapeshiftoss/caip": "^1.6.1",
     "@shapeshiftoss/thorchain": "^5.1.1",
     "@shapeshiftoss/types": "^1.17.0",
+    "abi-decoder": "^2.4.0",
```

Comment:
Is there a reason we aren't just using `ethers.js` for abi decoding purposes to avoid bringing in another package?

Reply:
It would also reduce the need for the extra `ABI.Item` types being spread throughout I would assume since ethers understands vanilla json arrays

Reply:
I thought there was a reason, but it was not valid. I've stripped out `abi-decoder` entirely in favour of `ethers` https://github.com/shapeshift/unchained/pull/267/commits/5606fc19cb7ae325866e20a8fa0d124f4330d5c0.